manifests/m/Microsoft/VisualStudioCode/Insiders/1.58.0/Microsoft.VisualStudioCode.Insiders.locale.en-US.yaml

```
@@ -1,6 +1,6 @@
 PackageIdentifier: Microsoft.VisualStudioCode.Insiders
-PackageName: Visual Studio Code Insiders
-PackageVersion: 1.58.1.e1161be268
+PackageName: Microsoft Visual Studio Code Insiders
+PackageVersion: 1.58.0
```

```suggestion
PackageName: Microsoft Visual Studio Code Insiders
PackageVersion: 1.58.0
```
Ahhh so it actually got Insider in the name, gotta remember to fix the PackageVersion then..